packages/app/app/containers/BestNewMusicCardContainer/BestNewMusicCardContainer.test.tsx

```diff
@@ -0,0 +1,80 @@
+import { waitFor } from '@testing-library/react';
+import pitchforkBnm from 'pitchfork-bnm';
+import { Deezer } from '@nuclear/core/src/rest';
+
+import { buildStoreState } from '../../../test/storeBuilders';
+import { mountedComponentFactory, setupI18Next } from '../../../test/testUtils';
+
+jest.mock('pitchfork-bnm');
+jest.mock('@nuclear/core/src/rest');
+
+describe('Dashboard container', () => {
+  beforeAll(() => {
+    setupI18Next();
+  });
+
+  beforeEach(() => {
+    const mockState = buildStoreState()
+      .withDashboard()
+      .build();
+
+    pitchforkBnm.getBestNewAlbums = jest.fn().mockResolvedValue(mockState.dashboard.bestNewAlbums);
+    pitchforkBnm.getBestNewTracks = jest.fn().mockResolvedValue(mockState.dashboard.bestNewTracks);
+
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    Deezer.getTopTracks = jest.fn().mockResolvedValue({ data: mockState.dashboard.topTracks });
+    
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    Deezer.mapDeezerTrackToInternal = jest.requireActual('@nuclear/core/src/rest').Deezer.mapDeezerTrackToInternal;
+  });
+
+  afterAll(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should render card album title', async () => {
```

Comment:
These four are basically the same exact test. If the snapshot is rendered as you think it should be rendered, there's no need to split it into 4 separate tests. Either put all the expects in a single test, or simply use the snapshot and remove the expects, since the snapshot will check the same things.

Reply:
Have consolidated the four expects into one test.